pyworkflow/pyworkflow/node.py

```diff
@@ -226,7 +226,55 @@ class PivotNode(ManipulationNode):
         'dropna': True,
         'margins_name': 'All',
         'observed': False
+    }
 
+    OPTION_TYPES = {
+        'index': {
+            "type": "column, grouper, array or list",
```

Comment:
The front-end uses `type` to figure out what kind of form field to render. If it's unknown (for now meaning anything other than `file` or `string`, it just skips it. After milestone 2 we need to sit down and define the possible parameter types.